[
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/6068689",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/62#issuecomment-6068689",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/62",
    "id": 6068689,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-06-01T19:22:15Z",
    "updated_at": "2012-06-01T19:22:15Z",
    "body": "You shouldn't change the package settings, but rather the user or project settings.\n\nIs your locatedb up to date?\nIs that the option used by sublimeclang in the end? Set debug_options to true, clear the cache and look in the python console."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/6075922",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/62#issuecomment-6075922",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/62",
    "id": 6075922,
    "user": {
      "login": "alnkpa",
      "id": 509141,
      "avatar_url": "https://secure.gravatar.com/avatar/a6b707ce7098064ff765ae1b7ac7ea4e?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "a6b707ce7098064ff765ae1b7ac7ea4e",
      "url": "https://api.github.com/users/alnkpa",
      "html_url": "https://github.com/alnkpa",
      "followers_url": "https://api.github.com/users/alnkpa/followers",
      "following_url": "https://api.github.com/users/alnkpa/following",
      "gists_url": "https://api.github.com/users/alnkpa/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/alnkpa/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/alnkpa/subscriptions",
      "organizations_url": "https://api.github.com/users/alnkpa/orgs",
      "repos_url": "https://api.github.com/users/alnkpa/repos",
      "events_url": "https://api.github.com/users/alnkpa/events{/privacy}",
      "received_events_url": "https://api.github.com/users/alnkpa/received_events",
      "type": "User"
    },
    "created_at": "2012-06-02T08:48:20Z",
    "updated_at": "2012-06-02T08:48:54Z",
    "body": "The problem was that I have user settings that I forgot about. Those overwrote the package one's, apparently."
  }
]
